src/dataset.ts

```diff
@@ -185,7 +188,7 @@ function checkMetadata(o: any) {
     }
 
     if ('authors' in o) {
-        if (typeof o.authors.length === undefined) {
+        if (!Array.isArray(o.authors)) {
             throw Error('"meta.authors" must be an array in dataset');
```

Comment:
Can we add string comprehension in here?

Reply:
You mean something like this?

```
            throw Error(`"meta.authors" must be an array in dataset, got ${o.authors}`);
```

Reply:
Error is better, but by string comprehension, I mean not enforcing that Authors need to be a list, but rather parse a string into a list ala bibtex

Reply:
I don't understand what you mean here. Transforming `"Doe, John and Jean, Michel"` into `["John Doe", "Michel Jean"]`? I would rather not go this way, if people want to use something based on bibtex they can write some code to create the corresponding JSON file.

Reply:
Sure, just a suggestion.